convert/src/label_studio_to_badgerdoc/models/label_studio_models.py

```diff
@@ -124,8 +124,8 @@ class ModelItem(BaseModel):
     predictions: List[int] = []
     data: Data
     meta: Meta = Meta()
-    created_at: Optional[str] = "2022-12-13T09:57:08.451845Z"
-    updated_at: Optional[str] = "2022-12-13T09:57:08.451845Z"
+    created_at: Optional[str] = ""
+    updated_at: Optional[str] = ""
     inner_id: int = 1
     total_annotations: int = 0
     cancelled_annotations: int = 0
```

Comment:
This and all attributes with default values should be rechecked:
https://github.com/epam/badgerdoc/pull/117#discussion_r1061264133

Reply:
Fixed